### DISCUSSION TITLE
Verify public key signature

### DISCUSSION THREAD

**Discussion by joatra** [krzyzanowskim/ObjectivePGP#206] [General]
Is it possible to verify public key signatures?

I want to do something like this:
```
[ObjectivePGP verify:signedPK withSignature:signature usingKeys:@[signerPK] passphraseForKey:nil error:nil];
```
However, i can't figure out how to get the signature from the public key. 
I've tried the following code, but it returns nil:
```
PGPPacket *packet = [partialKey signingKeyPacketWithKeyID: [signerPK keyID]];
```

**Comment by krzyzanowskim** [comment]
Hm... I'm affraid this is part of the private API. This is why you can't access signatures directly.
